katdal/ms_extra.py

```diff
@@ -1006,7 +995,7 @@ def write_rows(t, row_dict, verbose=True):
             if col_data.dtype.kind == 'U':
                 col_data = np.char.encode(col_data, encoding='utf-8')
             try:
-                t.putcol(col_name, col_data.T if casacore_binding == 'casapy' else col_data, startrow)
+                t.putcol(col_name, col_data, startrow)
                 if verbose:
```

Comment:
I would suggest using the else clause in the try-except-else-finally structure.

```python
try:
   t.putcol(...)
except RuntimeError as e:
   print(...)
else:
    if verbose:
       print(...)
```

Reply:
Done (plus restore the Happy Path).